android/src/facebook/TiFacebookModule.java

```diff
@@ -379,6 +379,17 @@ public String getAccessToken()
 		}
 		return "";
 	}
+	
+	@Kroll
+		.getProperty
+		@Kroll.method
+		public boolean getAccessTokenIsActive()
```

Comment:
Also make a method out of this. Note: Both properties and methods are "methods" in the native lands. On Android, you define them via `@Kroll.method` or `@Kroll.setProperty` / `@Kroll.getProperty`. On iOS, methods always have the argument suffix, e.g. `- (BOOL)isLoggedIn:(id)unused`, whereby properties do not have it, e.g. `- (NSString *)username`.